cmd/profilecli/main.go

```diff
@@ -65,11 +65,11 @@ func main() {
 	tsdbSeriesFiles := tsdbSeriesCmd.Arg("file", "tsdb file path").Required().ExistingFiles()
 
 	queryCmd := app.Command("query", "Query profile store.")
-	queryMergeCmd := queryCmd.Command("merge", "Request merged profile.")
-	queryMergeOutput := queryMergeCmd.Flag("output", "How to output the result, examples: console, raw, pprof=./my.pprof").Default("console").String()
```

Comment:
Output flag is used in both `merge` and `go-pgo`. We can move it to `addQueryMergeParams` which is also called by `addQueryGoPGOParams`.